src/framework/auth/services/auth.service.ts

```diff
@@ -21,8 +21,21 @@ import { NbAuthToken } from './token/token';
 @Injectable()
 export class NbAuthService {
 
+  private authEndPoints: string[] = ['login', 'register', 'request-pass', 'logout', 'token', 'refresh'];
+
+  // Authorization server urls - a way to make them available in http_interceptor or anywhere else
+  authUrls: string[] = [];
+
   constructor(protected tokenService: NbTokenService,
               @Inject(NB_AUTH_STRATEGIES) protected strategies) {
+    for (const strategy of this.strategies) {
```

Comment:
To be honest I don't like this idea of having this kind of state in the auth service. I would say that the public urls should not be configured by the strategy endpoints, but rather separately inside of the interceptor. This way we make it explicit.

Reply:
@nnixaa yes i agree with you.
I had a long time asking myself how to make the auth urls available in the interceptor so that it does not intercept them.
I choosed not to ask the user to declare twice the auth urls, once in the strategy descripton, once elsewhere so that it could be injected or available in the interceptor.

So you agree with the fact that nebular users declare twice the auth urls ?

Reply:
I guess we shouldn't call these `auth urls`, these are rather `public urls`, which means that they are not limited to the urls used for the auth. Thus there should be a way to provide this list anyway.

Reply:
I think I got it.
You are saying that in every case the user has to provide a list of public urls so that it is not very embarrassing to ask him to add auth urls to his list ?